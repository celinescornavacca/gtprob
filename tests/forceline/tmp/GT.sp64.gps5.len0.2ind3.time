41.63user 0.48system 0:42.43elapsed 99%CPU (0avgtext+0avgdata 283376maxresident)k
16inputs+8outputs (0major+139334minor)pagefaults 0swaps
