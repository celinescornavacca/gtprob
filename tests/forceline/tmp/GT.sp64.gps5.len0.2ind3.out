-612.861943636
